client/dist/style.css

```diff
@@ -68,3 +71,161 @@
 .read-more {
   color: #00A699;
 }
+
+/* Styles for Header.jsx */
+.header-container {
+  margin: 30px 0px 0px 50px;
+}
+
+.header-title {
+  font-weight: 600;
+  font-size: 24px;
+}
+
+.total-rating-div {
+  display: inline;
+}
+
+.star {
+  margin: 5px 0;
+  position: relative;
+  display: inline-block;
+  color: #00A699;
+  width: 0px;
+  height: 0px;
+  border-right: 10px solid transparent;
+  border-bottom: 7px solid #00A699;
+  border-left: 10px solid transparent;
+  transform: rotate(35deg);
+}
+
+.star:before {
+  border-bottom: 8px solid #00A699;
+  border-left: 3px solid transparent;
+  border-right: 3px solid transparent;
+  position: absolute;
+  height: 0;
+  width: 0;
+  top: -4.5px;
+  left: -6.5px;
+  display: inline-block;
+  content: '';
+  transform: rotate(-35deg);
+}
+
+.star:after {
+  position: absolute;
+  display: inline-block;
+  color: #00A699;
+  top: 0.3px;
+  left: -10.5px;
+  width: 0px;
+  height: 0px;
+  border-right: 10px solid transparent;
+  border-bottom: 7px solid #00A699;
+  border-left: 10px solid transparent;
+  transform: rotate(-70deg);
+  content: '';
+}
+
+.total-rating {
+  font-weight: bold;
+  margin-left: 10px;
+  font-size: 18px;
+}
+
+.num-reviews-div {
+  display: inline;
+  font-weight: bold;
+  margin-left: 40px;
+  font-size: 18px;
+}
+
+.num-reviews-text {
+  font-weight: normal;
+}
+
+/* Styles for Ratings.jsx */
+.ratings-container {
+  height: 15vh;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-areas:
+    "checkin communication"
+    "accuracy cleanliness"
+    "location value";
+  margin: 30px 650px 0px 50px;
+}
+
+.rating-container {
+  position: relative;
+}
+
```

Comment:
I gave you a demo of this, but to recap, if you add an inner div on `.rating-container`, call it `.rating-container-div` and give it the style. 
`  position: absolute;
  width: 100%;`
you can use relative positioning (e.g. left: 0% for rating-title) for the rating-elements letting them line up perfectly